app/code/Magento/Customer/Test/Mftf/ActionGroup/StorefrontCustomerLoginActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontCustomerLoginActionGroup">
```

Comment:
I like the fact that you want to introduce proper naming for actions, but if you want to do so - deprecate old one: `LoginToStorefrontActionGroup`.